engine/execution/computation/manager_test.go

```diff
@@ -111,10 +113,14 @@ func TestComputeBlockWithStorage(t *testing.T) {
 	programsCache, err := NewProgramsCache(10)
 	require.NoError(t, err)
 
+	eds := new(state_synchronization.ExecutionDataService)
+	eds.On("Add", mock.Anything, mock.Anything).Return(flow.ZeroID, nil)
```

Comment:
For all these mocked functions, i'm kind of tempted to use a non-zero id, and save the flow.ZeroID for error scenarios

Reply:
@Kay-Zee Are you suggesting that we actually compute and return the correct ID? I agree that this would be more ideal, but it means that we'd have to implement the ID computation logic (non-trivial) even though it never actually gets used for anything.

Reply:
maybe not the correct ID, just not the ZeroID